src/e2e-test/java/io/cdap/plugin/cloudsqlpostgresql/stepsdesign/CloudSqlPostGreSql.java

```diff
@@ -95,57 +117,57 @@ public void verifyTheCldMySqlInvalidPrivate() throws InterruptedException {
-            CdfCloudSqlPostGreSqlActions.enterConnectionName(CdapUtils.pluginProp("pSqlConnectionNameValid"));
-            CdfCloudSqlPostGreSqlActions.enterTableName(CdapUtils.pluginProp("pSqlTableNameCS"));
+            CdfCloudSqlPostGreSqlActions.enterReferenceName(CdapUtils.pluginProp("cloudPsqlReferenceNameValid"));
+            CdfCloudSqlPostGreSqlActions.enterConnectionName(System.getenv("Cloud_Psql_ConnectionName"));
+            CdfCloudSqlPostGreSqlActions.enterTableName(CdapUtils.pluginProp("cloudPsqlTableName"));
         } else if (property.equalsIgnoreCase("connectionName")) {
-            CdfCloudSqlPostGreSqlActions.enterReferenceName(CdapUtils.pluginProp("pSqlReferenceNameValid"));
-            CdfCloudSqlPostGreSqlActions.enterDatabaseName(CdapUtils.pluginProp("pSqlDatabaseName"));
-            CdfCloudSqlPostGreSqlActions.enterTableName(CdapUtils.pluginProp("pSqlTableNameCS"));
+            CdfCloudSqlPostGreSqlActions.enterReferenceName(CdapUtils.pluginProp("cloudPsqlReferenceNameValid"));
+            CdfCloudSqlPostGreSqlActions.enterDatabaseName(CdapUtils.pluginProp("cloudPsqlDbName"));
+            CdfCloudSqlPostGreSqlActions.enterTableName(CdapUtils.pluginProp("cloudPsqlTableName"));
         } else if (property.equalsIgnoreCase("tableName")) {
-            CdfCloudSqlPostGreSqlActions.enterReferenceName(CdapUtils.pluginProp("pSqlReferenceNameValid"));
-            CdfCloudSqlPostGreSqlActions.enterDatabaseName(CdapUtils.pluginProp("pSqlDatabaseName"));
-            CdfCloudSqlPostGreSqlActions.enterConnectionName(CdapUtils.pluginProp("pSqlConnectionNameValid"));
+            CdfCloudSqlPostGreSqlActions.enterReferenceName(CdapUtils.pluginProp("cloudPsqlReferenceNameValid"));
+            CdfCloudSqlPostGreSqlActions.enterDatabaseName(CdapUtils.pluginProp("cloudPsqlDbName"));
+            CdfCloudSqlPostGreSqlActions.enterConnectionName(System.getenv("Cloud_Psql_ConnectionName"));
         } else if (property.equalsIgnoreCase("jdbcPluginName")) {
-            CdfCloudSqlPostGreSqlActions.enterReferenceName(CdapUtils.pluginProp("pSqlReferenceNameValid"));
+            CdfCloudSqlPostGreSqlActions.enterReferenceName(CdapUtils.pluginProp("cloudPsqlReferenceNameValid"));
             SeleniumHelper.replaceElementValue(CdfCloudSqlPostGreSqlLocators.driverName, "");
-            CdfCloudSqlPostGreSqlActions.enterDatabaseName(CdapUtils.pluginProp("pSqlDatabaseName"));
-            CdfCloudSqlPostGreSqlActions.enterConnectionName(CdapUtils.pluginProp("pSqlConnectionNameValid"));
-            CdfCloudSqlPostGreSqlActions.enterTableName(CdapUtils.pluginProp("pSqlTableNameCS"));
+            CdfCloudSqlPostGreSqlActions.enterDatabaseName(CdapUtils.pluginProp("cloudPsqlDbName"));
+            CdfCloudSqlPostGreSqlActions.enterConnectionName(System.getenv("Cloud_Psql_ConnectionName"));
+            CdfCloudSqlPostGreSqlActions.enterTableName(CdapUtils.pluginProp("cloudPsqlTableName"));
         }
     }
```

Comment:
fail test if property is not one of the above values

Reply:
done